standup/script.js

```diff
@@ -0,0 +1,207 @@
+const searchInput = document.getElementById('user-search-input');
+const searchButtonElement = document.getElementById('search-button');
+const tableContainerElement = document.getElementById('table-container');
+const tableElement = document.createElement('table');
+tableElement.classList.add('user-standup-table');
+const tableBodyElement = document.createElement('tbody');
+
+const currentDateObj = new Date();
+const currentYearNum = currentDateObj.getFullYear();
+const daysInCurrentMonth = new Date(
+  currentYearNum,
+  currentDateObj.getMonth() + 1,
+  0,
+).getDate();
+const currentMonthName = currentDateObj.toLocaleString('default', {
+  month: 'long',
+});
+
+async function fetchUserData(username) {
+  const response = await makeApiCall(`${RDS_API_USERS}/${username}`);
```

Comment:
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/81404366/b7e8939b-91e4-4bf9-94a0-5a3cd5aedf51) 
as you can see the username can contain numbers as well as special characters. It is not best for searching. To address this, I think we have to make changes in the query in the db all together. 
I am approving it for now.  Please clarify this doubt once and create an issue for this if necessary,

Reply:
The issue ticket for this backend issue has been created.